commands/hugo.go

```diff
@@ -593,8 +605,15 @@ func copyStatic() error {
 // getDirList provides NewWatcher() with a list of directories to watch for changes.
 func getDirList() []string {
 	var a []string
+	var layoutPath string
+
+	if layoutDir != "" {
+		layoutPath = helpers.AbsPathifySimple(viper.GetString("LayoutDir"))
```

Comment:
Why AbsPathifySimple? These new dir flags should behave exactly as the others. If there is some issue with AbsPathify, then that should be addressed in a separate issue.